cli.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"strings"
+
+	"github.com/mendersoftware/log"
+	"github.com/mendersoftware/mender/installer"
+	"github.com/mendersoftware/mender/system"
+
+	"github.com/pkg/errors"
+	"gopkg.in/urfave/cli.v2"
```

Comment:
You need to add this dependency with `govendor`

Reply:
And most importantly: using v2 will break compatibility with Debian, which will lead to a) our package no being updated in upstream or b) us assuming even more maintenance work (adding such dep in Debian).

Is there a good reason for the upgrade?

Reply:
The dependency is still missing. Nothing in the pipeline will run if we are not able to compile the target.

Reply:
Done